### PR TITLE
TLZ strong Leagcy update site code change

### PR DESCRIPTION
**Pull request by salichon**

### CODE REVIEW

File: install/recorders.csv [hunk 1]
```diff
@@ -1018,8 +1018,7 @@ Kinemetrics,FBA-ES-T-BASALT,BASALT,620,INGS,20,280,,0,0,2015-06-30T05:15:00Z,201
 Kinemetrics,FBA-ES-T-BASALT,BASALT,620,GKBS,20,303,,0,0,2020-09-03T21:13:00Z,9999-01-01T00:00:00Z
 Kinemetrics,FBA-ES-T-BASALT,BASALT,621,NNZ,20,0,,0,0,2012-01-05T23:10:00Z,2013-01-17T21:00:00Z
 Kinemetrics,FBA-ES-T-BASALT,BASALT,621,INZ,20,0,,0,0,2013-06-12T00:10:00Z,2015-05-28T06:00:00Z
-Kinemetrics,FBA-ES-T-BASALT,BASALT,621,TLZ,20,0,,0,0,2016-02-12T01:30:01Z,2020-01-15T02:12:00Z
-Kinemetrics,FBA-ES-T-BASALT,BASALT,621,TLZ,21,0,,0,0,2020-01-15T02:16:00Z,2020-07-02T02:31:00Z
+Kinemetrics,FBA-ES-T-BASALT,BASALT,621,TLZ,21,0,,0,0,2016-02-12T01:30:01Z,2020-01-15T02:12:00Z
```
Comment (mnaguit) [inline install/recorders.csv:1021]: Should the end date of Basalt be `2020-07-02T02:31` at LC 21?
Noting that data exists from January to July, up to when the next recorder (Titan) was installed `2020-07-05`.

Sample data randomly checked: 
```
aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.032/TLZ.NZ/
2020-02-06 20:44:00    7744512 2020.032.TLZ.21-BNE.NZ.D
2020-02-06 20:43:57    7787520 2020.032.TLZ.21-BNN.NZ.D
2020-02-06 20:43:56    7045120 2020.032.TLZ.21-BNZ.NZ.D

```
```
aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.099/TLZ.NZ/
2020-04-14 05:54:09    8551424 2020.099.TLZ.21-BNE.NZ.D
2020-04-14 08:52:29    8388608 2020.099.TLZ.21-BNN.NZ.D
2020-04-14 05:54:09    7639552 2020.099.TLZ.21-BNZ.NZ.D
2020-04-14 08:52:29   28896256 2020.099.TLZ.21-HNE.NZ.D
2020-04-14 08:52:29   27369984 2020.099.TLZ.21-HNN.NZ.D
2020-04-13 19:49:38   27885056 2020.099.TLZ.21-HNZ.NZ.D
```
```
aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.184/TLZ.NZ/
2020-07-06 23:33:45    4366848 2020.184.TLZ.21-HNE.NZ.D
2020-07-06 23:33:46    4312064 2020.184.TLZ.21-HNN.NZ.D
2020-07-07 02:30:45    4393984 2020.184.TLZ.21-HNZ.NZ.D
```

File: install/streams.csv [hunk 2]
```diff
@@ -2013,10 +2013,10 @@ TLED,20,200,true,false,true,2010-09-05T05:00:02Z,2012-02-21T01:00:00Z
...
 TLZ,10,100,false,false,false,2009-05-07T01:00:00Z,9999-01-01T00:00:00Z
 TLZ,20,1,false,false,false,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
-TLZ,20,50,false,false,false,2009-05-07T01:00:00Z,2020-01-15T02:12:00Z
-TLZ,20,200,false,false,true,2009-05-07T01:00:00Z,2020-01-15T02:12:00Z
-TLZ,21,50,false,false,false,2020-01-15T02:16:00Z,2020-06-16T00:00:00Z
-TLZ,21,200,false,false,true,2020-01-15T02:16:00Z,2020-07-02T02:31:00Z
+TLZ,20,50,false,false,false,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
+TLZ,20,200,false,false,true,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
+TLZ,21,50,false,false,false,2016-02-12T01:30:01Z,2020-06-16T00:00:00Z
+TLZ,21,200,false,false,true,2016-02-12T01:30:01Z,2020-07-02T02:31:00Z
```
Comment (mnaguit) [inline install/streams.csv:2019]: It looks like 200hz triggered ended a bit earlier.
200hz continuous seems to start on 2020.040 (Feb9).
```
murieln@ubuntu:~/Desktop/Shared/sh$ ./list_miniseed.sh TLZ 2020 040 050 | grep HNN | sed -e 's/"//g' -e 's/Key//' -e 's/://'
  miniseed/2020/2020.040/TLZ.NZ/2020.040.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.040/TLZ.NZ/2020.040.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.041/TLZ.NZ/2020.041.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.042/TLZ.NZ/2020.042.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.043/TLZ.NZ/2020.043.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.044/TLZ.NZ/2020.044.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.045/TLZ.NZ/2020.045.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.046/TLZ.NZ/2020.046.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.047/TLZ.NZ/2020.047.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.048/TLZ.NZ/2020.048.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.049/TLZ.NZ/2020.049.TLZ.21-HNN.NZ.D
  miniseed/2020/2020.050/TLZ.NZ/2020.050.TLZ.21-HNN.NZ.D
```

**Review by mnaguit — CHANGES_REQUESTED** [review]
As per comments in relevant `csv `files

File: install/streams.csv [hunk 3]
```diff
@@ -2013,10 +2013,10 @@ TLED,20,200,true,false,true,2010-09-05T05:00:02Z,2012-02-21T01:00:00Z
 TLZ,10,1,false,false,false,2009-05-07T01:00:00Z,9999-01-01T00:00:00Z
 TLZ,10,100,false,false,false,2009-05-07T01:00:00Z,9999-01-01T00:00:00Z
 TLZ,20,1,false,false,false,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
-TLZ,20,50,false,false,false,2009-05-07T01:00:00Z,2020-01-15T02:12:00Z
-TLZ,20,200,false,false,true,2009-05-07T01:00:00Z,2020-01-15T02:12:00Z
-TLZ,21,50,false,false,false,2020-01-15T02:16:00Z,2020-06-16T00:00:00Z
-TLZ,21,200,false,false,true,2020-01-15T02:16:00Z,2020-07-02T02:31:00Z
+TLZ,20,50,false,false,false,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
+TLZ,20,200,false,false,true,2009-05-07T01:00:00Z,2016-02-12T01:30:00Z
```
Comment (mnaguit) [inline install/streams.csv:2017]: If we want it very specific, end date looks like around `2016-02-12T01:25:00`
And data for LC 21 starts at `01:25:12` based on the data plot on `2016.043`

**Review by mnaguit — CHANGES_REQUESTED** [review]

File: install/recorders.csv (hunk 1 above)
Comment (salichon) [reply install/recorders.csv:1021]: of course!

File: install/streams.csv (hunk 3 above)
Comment (salichon) [reply install/streams.csv:2017]: yeah the delta entry was  rounded apparently

Comment (salichon) [reply install/streams.csv]: will clean that up :)

**Review by salichon — COMMENTED** [review]
mkign the changes

**Comment by mnaguit** [comment]
Hey @salichon , thanks a lot for making the changes in this PR.
Just one more comment, hopefully the last:

- For the switch from LC 21 200hz triggered to LC 21 200hz continuous:
   - from the archive, 200hz continuous starts coming in on `2020.040` (9 February)
   - this however does not align with the recorder swap (from Basalt 621 to Titan 1241)
```
aws s3 ls --profile sod_readrole s3://geonet-archive/miniseed/2020/2020.040/TLZ.NZ/
...
2020-02-14 21:02:57    2224640 2020.040.TLZ.21-HNE.NZ.D
2020-02-14 21:02:57    2014208 2020.040.TLZ.21-HNN.NZ.D
2020-02-14 21:02:56    1993216 2020.040.TLZ.21-HNZ.NZ.D

```

What do you think?

**Comment by salichon** [comment]
Heya @mnaguit  all good i agree  the 200Hz swith occurat 2020.o40  indeed  instead of the dtatlogger switch (not related)
I assumne the date was chosen for convenienece :) 
I ll push that change

Besides i  identified oddites in 2020 (2020.187/188 for which the loc code is 20 instead of 21 - Wayyyyy after the site configuration change apparently.)
> ls /geonet/seismic/2020/NZ/TLZ/HNZ.D/ | sort -n
> **NZ.TLZ.20.HNZ.D.2020.187**
>**NZ.TLZ.20.HNZ.D.2020.188**
> NZ.TLZ.21.HNZ.D.2020.040
> NZ.TLZ.21.HNZ.D.2020.041
...

I ll add them in the datax PR  if not ther already :)

**Review by mnaguit — APPROVED** [review]
Okaydokay, i'll approve this now 👍

**Pull request merged**